src/test/java/org/opensearch/security/SecurityAdminTests.java

```diff
@@ -89,8 +88,7 @@ public void testSecurityAdminInvalidCert() throws Exception {
         argsAsList.add(String.valueOf(clusterInfo.httpPort));
         argsAsList.add("-cn");
         argsAsList.add(clusterInfo.clustername);
-        argsAsList.add("-cd");
-        argsAsList.add(new File("src/test/resources/").getAbsolutePath());
+        addDirectoryPath(argsAsList, TEST_RESOURCE_ABSOLUTE_PATH);
```

Comment:
Same comment on PR scope, looks like these test changes are not required for this PR either.

Reply:
We had many tests that manipulate the file system, under the OpenSearch test runner these paths have changed.  I mitigated as much as possible to prevent code change but some tests needed individual attention